### DISCUSSION TITLE
Snow water equivalent output

### DISCUSSION THREAD

**Discussion by rshresth1** [iiasa/CWatM#122] [Q&A] [answered]
Hello,

I have been working to resolve the rapid snowmelt issue when using snowmelt_radiation = True. I added albedo feedback to the radiation calculation, and the results look much better. 
I would like to look at the snow water equivalent results before submitting a pull request, but when I tried to output Out_Map_Daily = SnowWaterEquivalent, I get an error: "No values in" SnowWaterEquivalent on date: XXX".  Will appreciate help in resolving this issue.

Thank 
Raj

**Comment by mikhailsmilovic** [comment, marked as answer]
`SnowCover` is the variable for the depth of moisture [metres] in the snowpack.
This is discussed further here: https://github.com/iiasa/CWatM/discussions/97
